### DISCUSSION TITLE
audi q7 2017

### DISCUSSION THREAD

**Discussion by olegullikmejdell** [Mr-MIBonk/M.I.B._More-Incredible-Bash#113] [Q&A]
Hi,
I have a Audi Q7 2017.
I have mhi2_er_aug22_p3241, MU-sofware: 0678, I can see this is not tested in the MHI2 M.I.B. patch FW compatibility table.
Will it work for me?

thanks and regards!

**Comment by harman-f** [comment]
Sure, will work

**Reply by olegullikmejdell** [reply]
Thanks :) Ensure that a external power is connected to the car during any flash or programming process... So I need the charging cable connected to the q7 etron and be sure that not the engine is on? Only in battery mode?

**Reply by harman-f** [reply]
YES
